Applications/JupyterHub.qmd

```diff
@@ -3,72 +3,47 @@ title: "JupyterLab"
-They provide a user-friendly interface, integrating well with services like Sentinel Hub and openEO. 
-Each notebook has direct access to the EO Data repository, and example notebooks are available to facilitate data interaction. 
-To handle tool dependencies and library versions, multiple notebook kernels are provided and regularly updated with the provided samples. 
-This eliminates the need for users to handle dependency installations, enabling immediate prototyping without technical obstacles.
+Upon clicking the "ACCESS JUPYTERLAB" button, users will be directed to the login window. Please use Copernicus Data Space Ecosystem credentials to log in.
 
 
-## Notebook flavors
+## Server Options
 
-When you log in to JupyterLab you are presented with a choice of 3 flavors of Jupyter instances: Small, Medium and Large.
-The size of the instance is determined by number of resources available to the notebook kernels run by the user - CPU cores and memory.
-All flavors are suitable for performing typical tasks and will be capable of running all samples provided in /samples folder.
-To ensure fair use of resources by the CDSE users, it is recommended to start with the Small flavor and switch to a bigger only when you experience issues with kernel crashing due to the lack of available memory.
+Once logged in to JupyterLab, users are offered a choice of 3 flavors of Jupyter instances: Small, Medium and Large.
+The size of the instance is based on how many resources, like CPU cores and memory, are allocated to the notebook kernels.
+All flavours are suitable for performing simple tasks and can run all samples provided in the samples folder. For fair resource usage, it is suggested to start with the "Small" flavour and switch to a larger one if kernel crashes is encounted caused by insufficient memory.
 
 ![](_images/Flavors_captcha.png)
 
 ## JupyterLab User Interface
 
-Once you have successfully signed in, you will be presented with a launcher that offers various Python environments to work in, including Python 3, Geo science, OpenEO, and Sentinel Hub. Each environment is equipped with specific Python packages tailored to different requirements. You can choose to run your code either in a notebook or a console, depending on your preference. Additionally, the launcher provides options to create text files, markdown files, or Python files, allowing you to work with different types of documents as needed.
+After signing in, users will see a launcher with Python environments such as Python 3, Geo Science, OpenEO, and Sentinel Hub. Each environment is equipped with specific Python packages tailored to various requirements. Users can run their code in a notebook or a console, depending on their preference. Additionally, options are provided to create text files, markdown files, or Python files, allowing users to work with different types of documents as needed.
```

Comment:
can we link to a place in documentation which helps users pick between the environments?

Reply:
These kernels are created based on a couple of APIs; however, the different kernels can be used for an API or none of these. So, I'm not sure where I can link these. However, a page/section that describes these kernels or env. can be included within this. 


@KanerLev @Slegersj